src/ventus/wm/window.js

```diff
@@ -486,8 +489,23 @@ function(Emitter, Promise, View, WindowTemplate) {
 		},
 
 		move: function(x, y) {
-			this.x = x;
-			this.y = y;
+			var max_x = window.innerWidth - this._wm.margin.right;
+			var max_y = window.innerHeight - this._wm.margin.bottom;
```

Comment:
From what I can tell is Ventus doesn't allow for creating it inside of just a container at this time. It is automatically added into the DOM as an instance of the `WindowManager` class without any third party intervention. I may have just overlooked it however. That is why it is using the window's size.